pipeline/authn/authenticator_jwt.go

```diff
@@ -109,3 +111,29 @@ func (a *AuthenticatorJWT) Authenticate(r *http.Request, session *Authentication
 
 	return nil
 }
+
+func (a *AuthenticatorJWT) tryEnrichResultErr(token string, err *herodot.DefaultError) *herodot.DefaultError {
+	t, _ := jwt.ParseWithClaims(token, jwt.MapClaims{}, nil)
+	if t == nil {
+		return err
+	}
+	claims, ok := t.Claims.(jwt.MapClaims)
+	if !ok {
+		return err
+	}
+	var claimKeyPairs []string
+	for k, v := range claims {
+		// print just root content of the JWT, skip nested objects
+		if _, ok := v.(map[string]interface{}); ok {
+			continue
+		}
+		if floatVal, ok := v.(float64); ok && v == float64(int64(floatVal)) {
```

Comment:
This checks if v is a float, then converts it to an int, and then back to a float, and then back to an int. Do we really need this? Why no tjust `fmt.Sprintf("%v", v)`?

Reply:
this isn't really the pretties thing, but i wanted to print the full integer value and not something like this `iat=1.616678102e+09`(this is the result using `fmt.Sprintf("%v", v)`)

Reply:
The easiest way to do that is to format the output as JSON then, so:

```
var b bytes.Buffer
json.Encode(&b, claims)
err.WithDetail("claims", b.String())
```

which would also make it easier to process in log systems because it is a standard format